src/scripts/components/MessageBox.jsx

```diff
@@ -58,6 +58,13 @@ class MessageBox extends React.Component {
     }
   }
 
+  handleFocus = () => {
+    const inboxThreadActive = this.props.messageBoxType ? document.getElementById(`convo__footer__scroll__${this.props.messageBoxType}`) : null;
```

Comment:
@marymhart I think this approach is creating too tight a dependency between `rhinostyle` and `rhinofront` since the focus will ONLY work properly based on IDs that are external to this library. Ideally the UI library would be completely agnostic to the app that uses it.

I think a better approach would be to create a `onFocus` prop that can get passed to the `MessageBox` component. That way you can control your scrolling behavior from `rhinofront` and we avoid sprinkling pieces of outside context into our UI library. 

Let me know what you think and if you want to chat about it further.

Reply:
@juanfabrega yes that makes sense! I'll update the PRs